cost/aws/rds_instance_cloudwatch_utilization/rds_instance_cloudwatch_utilization.pt

```diff
@@ -0,0 +1,411 @@
+name "AWS Rightsize RDS Instances"
+rs_pt_ver 20180301
+type "policy"
+short_description "Gathers AWS CloudWatch data for RDS instances on 30 day intervals. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/rds_instance_cloudwatch_utilization/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
use:  Finds AWS RDS Instances using CPU threshold and resizes them after approval.